tools/jenkins/run_performance.sh

```diff
@@ -34,4 +34,5 @@ set -ex
 # Enter the gRPC repo root
 cd $(dirname $0)/../..
 
-tools/run_tests/run_performance_tests.py -l c++ node ruby csharp python --netperf --category smoketest
+tools/run_tests/run_microbenchmark.py -c summary --diff_perf origin/$ghprbTargetBranch
```

Comment:
ok, so if this really disables non-C/C++ scenarios, I am against it.
It is fine to reduce number of scenarios we are running if there's a problem with the tests running for too long (I think majority of them is C/C++ anyway) but this suite is our only protection against breaking performance tests in master (and as a side effect this works as a de-facto stress test for wrapped languages and has lead to discovering issue in the past) - so removing it seems really off.

I think microbenchmarks are great and so is getting a clear signal about regression on a PR, but they should be added as a new Jenkins job. Killing an existing test suite for OTHER languages in order to save a bit of time setting up a new job is not the right way to do things.